### DISCUSSION TITLE
part-ii/2017-61/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#146] [Question comments]
Part II 2017 Further Complex Methods: Paper 1, Section I, E 

https://questions.tripos.org/part-ii/2017-61/

**Comment by acornnroca** [comment]
<details>
<summary>Principal Value</summary>

$$i \pi$$ for $$n=1$$ by using indentation lemma, $$\infty$$ for $$n \geq 2$$ taking a contour that's a circle around the origin

</details>
